### ISSUE TITLE
Single Trailing Newline Rule

### ISSUE THREAD

**Issue by rajohns08** [realm/SwiftLint#820]
This rule doesn't trigger if the file has 0 trailing newlines. It does trigger if it has 2 or more trailing newlines, though.

**Comment by jpsim** [comment]
Sorry for the delay in getting back to you. I couldn't reproduce this with the information provided.

```shell
$ mkdir test
$ cd test
$ touch test.swift
$ swiftlint
Linting Swift files in current working directory
Linting 'test.swift' (1/1)
test.swift:1: warning: Trailing Newline Violation: Files should have a single trailing newline. (trailing_newline)
Done linting! Found 1 violation, 0 serious in 1 file.
```

Please reopen a new issue with steps to reproduce if you'd like us to take another look at this.
